api/envoy/extensions/filters/http/local_ratelimit/v3/local_rate_limit.proto

```diff
@@ -107,4 +107,10 @@ message LocalRateLimit {
   // one to rate limit requests on a per connection basis.
   // If unspecified, the default value is false.
   bool local_rate_limit_per_downstream_connection = 11;
+
+  // Defines the standard version to use for X-RateLimit headers emitted by the filter:
+  //
+  // Disabled by default.
+  common.ratelimit.v3.XRateLimitHeadersRFCVersion enable_x_ratelimit_headers = 12
```

Comment:
nit: maybe just `ratelimit_headers_version` or `x_ratelimit_headers_version`?

Reply:
Ok, I will changed to `x_ratelimit_headers_version`.

Reply:
Some thoughts, I think it is worth considering since I want to align with `ratelimit`'s `enable_x_ratelimit_headers`. Maybe we can keep the name unchanged?